src/components/MessageInput/MessageInput.tsx

```diff
@@ -485,6 +504,8 @@ export const MessageInputWithContext = <
               getUsers={getUsers}
               openAttachmentPicker={openAttachmentPicker}
               openCommandsPicker={openCommandsPicker}
+              openFilePicker={pickFile}
```

Comment:
feel like we should keep the function name and the prop name the same so `pickFile={pickFile}` or `openFilePicker={openFilePicker}` depending on what the function actually does

Reply:
Ok makes sense!!